Dockerfile.agent

```diff
@@ -0,0 +1,33 @@
+FROM docker.elastic.co/elastic-agent/elastic-agent:sha256-fff74ac535403af9307ed6f93aad71ec95866bf6f1ec2da1bd7966e6bc17ae72
```

Comment:
Let's add a comment or something at the top of this, so that it's clear to anyone looking at it that this is for internal/experimental purposes only.

Reply:
The real `elastic-agent-service` container is based off of Wolfi, best to use `elastic-agent-wolfi` here.

Reply:
`elastic-agent-wolfi` does not seem to have `apt` and a couple other things installed.

Since this file is just for local development (which is also probably a temporary file, we'll find something more suitable hopefully), I think it's fine to use non-wolfi image.

I'm gonna update this line to point to latest 9.0.0-SNAPSHOT docker image